corehq/apps/app_manager/models.py

```diff
@@ -2095,12 +2095,22 @@ class DefaultCaseSearchProperty(DocumentSchema):
     default_value = StringProperty()
 
 
+class CaseSearchLabel(DocumentSchema):
+    label = DictProperty(default={'en': 'Search All Cases'})
```

Comment:
open to suggestions if this should be text or anything else so that the code references read well `search_config.search_label.text` instead of `search_config.search_label.label`

Reply:
`label` sounds good to me